tests/common/plugins/conditional_mark/__init__.py

```diff
@@ -126,29 +139,31 @@ def load_basic_facts(session):
 
     return results
 
-
-def find_longest_match(nodeid, case_names):
-    """Find the longest match of the given test case name in the case_names list.
+def find_longest_matches(nodeid, conditions):
+    """Find the longest matches of the given test case name in the case_names list.
```

Comment:
Behavior of this function is now lightly different. Need to update the docstring as well.